### DISCUSSION TITLE
Recommended way of hooking up external power

### DISCUSSION THREAD

**Discussion by bthitney** [gopro/labs#116] [Q&A] [answered]
Thanks to a previous discussion I’ve been able to make use of the QR codes, specifically for star trails and night time lapse. As an example, I ran this command tonight: !21:35NmST!S!06:15E

It needs external power to capture star trails that long. I have a Volta and also a decent USB bank. Even with the Volta, I seem to have a problem with it occasionally locking up when connected or, while less problematic, constantly turning on, realizing it has some time until it needs to start, shut off, and then turn back on again/repeat. With the USB-C battery bank, it seems a bit more iffy and sometimes seems to just freeze (screen on, and showing the last frame of whatever was on screen at the moment) and need to hard power cycle to get it back.

After scouring for info, it looks like one recommendation is to run without a battery. Just wanted to see if there are recommended approaches or troubleshooting I can do to make this more reliable.

**Comment by dnewman-gpsw** [comment, marked as answer]
There may be an issue using Volta with the Labs sleep command, so in this particular case why not just use the scheduled capture feature from the camera's menus, which is completely compatible with Volta.  You will get many many hours of operation for a star trails.  Using external battery or external power with no external battery, is really for multi-day events.

**Reply by bthitney** [reply]
Thanks! If I schedule the capture in the menu, I can’t specify a duration longer than 3 hours. I guess one solution is to let it go as long as it can with Volta, which would probably be about 7 hours. Or perhaps it is possible to combine  a scheduled capture and QR code for specific situations, but that’s probably overthinking things.

**Reply by dnewman-gpsw** [reply]
Just let the Volta run flat, as combining might not work out so well.

**Reply by bthitney** [reply]
Thanks! That makes sense and is the easiest solution. As an aside — and no solution here is needed — I noticed in all this testing that my Hero 10 works fine off my USB C battery pack, but my Hero 11 always locks up when connecting it. The battery is for my bike lights but doubles as nice way to keep the cam going on long rides. Just thought I’d mention.
